webpack.mix.js

```diff
@@ -1,17 +1,17 @@
 const mix = require('laravel-mix');
 
-mix.js('resources/js/app.js', 'public/js')
-    .js('resources/js/createSubject.js', 'public/js')
-    .js('resources/js/trainee_calendar.js', 'public/js');
+mix.js('resources/js/createSubject.js', 'public/js/createSubject.js')
+    .js('resources/js/trainee_calendar.js', 'public/js/trainee_calendar.js')
```

Comment:
```suggestion
mix.js('resources/js/create_subject.js', 'public/js')
    .js('resources/js/trainee_calendar.js', 'public/js')
```
đặt tên file js snake case đi chị :v